mocks/controllers/BasicAuthenticationController.go

```diff
@@ -40,10 +43,14 @@ func (_m *BasicAuthentificationController) BasicLoginHandler(_a0 http.ResponseWr
 }
 
 // Logout provides a mock function with given fields: _a0, _a1
-func (_m *BasicAuthentificationController) Logout(_a0 http.ResponseWriter, _a1 *http.Request) (interface{}, httperrors.HTTPError) {
+func (_m *BasicAuthenticationController) Logout(_a0 http.ResponseWriter, _a1 *http.Request) (interface{}, httperrors.HTTPError) {
 	ret := _m.Called(_a0, _a1)
 
 	var r0 interface{}
+	var r1 httperrors.HTTPError
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.